features/bevragen/persoon/adressering/aanhef/summary.feature

```diff
@@ -1,16 +1,129 @@
+# language: nl
+
 Functionaliteit: Als gemeente wil ik de juiste en consistente briefaanhef in communicatie naar burgers
   De aanhef is een standaardformulering waar mee de overheid zich tot de geadresseerde richt in de vorm van een introducerende zin in een brief of een email.
 
-Rule: De aanhef van een persoon wordt samengesteld door de voorletters, voorvoegsels en geslachtsnaam te combineren. 
```

Comment:
Is het niet handiger om deze rule te splitsen voor persoon zonder partner en persoon met (ex-)partner? Dan is het direct duidelijk welke rules er in die situatie gelden. De rule voor persoon zonder partner zou dan als volgt uit kunnen zien:

```
Rule: de aanhef voor een persoon zonder partner wordt als volgt samengesteld: Geachte GA VV GN, waarbij
        - GA wordt bepaald aan de hand van het geslacht van de persoon
        | geslacht| GA |
        | M|heer|
        | V|vrouw|
        | O|VL (voorletters)|
       - VV is de voorvoegsel geslachtsnaam van de persoon
       - GN is de geslachtsnaam van de persoon
 ```

Reply:
Voorstel overgenomen.  Rule voor persoon zonder partner  expliciet opgenomen en scenario's daaronder geplaatst.